twins/mocks/service.go

```diff
@@ -42,11 +40,10 @@ func CreateDefinition(names []string, subtopics []string) twins.Definition {
 }
 
 // CreateSenML creates SenML record array
-func CreateSenML(n int, bn string) []senml.Record {
+func CreateSenML(n int) []senml.Record {
```

Comment:
I think that that it is better to precreate array
like var recs[n]senml.Record and then initialize each element